### ISSUE TITLE
VoiceOver does not interrupt the old slider value of Interval Tool "Region" when moving quickly

### ISSUE THREAD

**Issue by marlitas** [phetsims/center-and-variability#634]
From: https://github.com/phetsims/center-and-variability/issues/631

To replicate:

1. Load the variability screen and check the Interval Tool checkbox.
2. Make sure voiceover is turned on a tab to the component named "Interval Region". You will traverse through the two slider handles on the tool first.
3. Using arrow keys quickly move the region along the field. This can be done with quick independent keystrokes or press and hold.
4. Notice that VoiceOver will not interrupt the old value to read the new value.

I did more testing on this and noticed that VoiceOver is doing similar behavior with the other sliders in the simulation. It is harder to tell since the values are considerably shorter, but the delay can be noticed especially when moving through decimal points.

@jessegreenberg tested with NVDA and was unable to replicate. I also tested with basic HTML sliders and I believe I can identify the same issue there, although it feels harder to be certain. With the current information we have this seems like a VoiceOver specific bug, but would like QA to keep an eye on it as CAV goes through the testing process. This should definitely get flagged if we are able to replicate with other screen reader software beyond VoiceOver as well.

**Comment by Nancy-Salpepi** [comment]
I can replicate this behavior in dev-6 with VO. 
I tried published GFL:B and the behavior was inconsistent--sometimes it interrupted and sometimes it didn't.

**Comment by KatieWoe** [comment]
On Win 11 Chrome + Jaws the reading was consistently interrupted. Most of the time when the arrow was released the new values were read. However, sometimes after holding the arrow down, it doesn't say the new value.
